package.json

```diff
@@ -11,14 +11,20 @@
   },
   "dependencies": {
     "@next/font": "13.1.2",
+    "@svgr/cli": "^6.5.1",
     "eslint": "8.32.0",
     "eslint-config-next": "13.1.2",
+    "file-loader": "^6.2.0",
```

Comment:
Where do you use that?

Reply:
I had some problems, so i just tried something, there is no usage...should i remove it ?